frontend/assets/style/components/_forms.scss

```diff
@@ -389,7 +389,7 @@ p.error {
       height: 6px;
       background-color: $background_0;
       margin-top: 1px;
-      top: 50%;
+      top: 10px;
```

Comment:
radio-box UI gets broken when its sentence becomes multiple lines. This is the fix for that bug.